components/link/Link.js

```diff
@@ -15,11 +15,14 @@ class Link extends PureComponent {
       className,
     );
 
+    const ChildrenWrapper = icon ? 'span' : Fragment;
     const Element = element || 'a';
```

Comment:
Just for improvement: Maybe you could just add `element` to `defaultProps` and assign 'a' to it